scenario_editor/src/Matrix/Matrix.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './Components.css';
```

Comment:
Make sure to fix all of these naming issues and push it to the repo so other people don't have to rename them manually